bigquery/google/cloud/bigquery/schema.py

```diff
@@ -41,9 +41,13 @@ def __init__(self, name, field_type, mode='NULLABLE', description=None,
         self.field_type = field_type
         self.mode = mode
         self.description = description
-        self.fields = fields
+        self.fields = None if(fields is None) else tuple(fields)
```

Comment:
I'd say that we should go ahead and make `fields=()` the default.  We would then need to adjust the docstring, and fix any unit tests which rely on having `fields` be `None`.